persistence/src/main/scala/com/mesosphere/usi/persistence/KVRepositoryImpl.scala

```diff
@@ -0,0 +1,58 @@
+package com.mesosphere.usi.persistence
+
+import akka.{Done, NotUsed}
+import akka.stream.scaladsl.Source
+import com.mesosphere.usi.core.models.{PodRecord, ReservationRecord}
+import com.mesosphere.usi.persistence.storage.{KVStorage, Serialization}
+import SerializationFormats.{javaSerializationPodRecord, javaSerializationReservationRecord}
+
+import scala.concurrent.Future
+
+class KVRepositoryImpl(storage: KVStorage) extends Repository {
+
+  private def podRecordSeqNr2Bytes(seqNr: Long): Array[Byte] = {
+    s"podRecord-$seqNr".getBytes()
+  }
+
+  private def reservatonRecordSeqNr2Bytes(seqNr: Long): Array[Byte] = {
+    s"reservationRecord-$seqNr".getBytes()
+  }
+
+  override def storePodRecord(record: PodRecord): Future[Done] = {
+    val value = Serialization.toBinary(record)
+    val key = podRecordSeqNr2Bytes(record.sequenceNr)
+    storage.write(key, value)
+  }
+
+  override def storeReservationRecord(record: ReservationRecord): Future[Done] = {
+    val value = Serialization.toBinary(record)
+    val key = reservatonRecordSeqNr2Bytes(record.sequenceNr)
+    storage.write(key, value)
+  }
+
+  override def podRecords(fromSequenceNr: Long, toSequenceNr: Long): Source[PodRecord, NotUsed] = {
```

Comment:
what are those sequence numbers with pod records? where do they come from? I don't know how we identify individual records but I don't think it'll be a `Long`.